backend.py

```diff
@@ -161,7 +162,7 @@ def getPrepSequence(ans, interactive):
 
 def getMainRepoSequence(ans, repos):
     seq = []
-    seq.append(Task(repository.installFromRepos, lambda a: [repos] + [a.get('mounts'), a.get('kernel-alt'), a.get('linstor-version')], [],
+    seq.append(Task(repository.installFromRepos, lambda a: [repos] + [a.get('mounts'), a.get('kernel-alt'), a.get('linstor-version'), a.get('xapi-version')], [],
```

Comment:
Same comment as Yann in upgrade.py: I'd put the more easily upstreamable first, here xapi-version before kernel-alt.

Reply:
Will be handled later.